test/service/StoreService.test.js

```diff
@@ -127,7 +131,7 @@ describe('StoreService', () => {
 			expect(store).toBeDefined();
 
 			const reducerKeys = Object.keys(store.getState());
-			expect(reducerKeys.length).toBe(28);
+			expect(reducerKeys.length).toBe(29);
```

Comment:
There  should be a specific test for the 'navigationRail' reducer, too.

```suggestion
			expect(reducerKeys.includes('navigationRail')).toBeTrue();
```